pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -27,6 +27,8 @@ Welcome to EVE!
             dump-stacks
             dump-memory
             memory-monitor-update-config
+            psi-collector-start
+            psi-collector-stop
```

Comment:
for other commands we use on/off as a parameter - f.e. https://github.com/lf-edge/eve/pull/4141/files#diff-a3b42769797ef2e752af884f98fd0877b6034d918bae243c3f165d7d710987a1L25

(also this comment comes from the guy who did not use on/off himself but start/stop)

Reply:
In the case of remote access, on/off looks fine. You either turn the access on or off. It's like a property. In the case of this tool, you have to start and stop it. It's a process, not a property. But I like the idea of adding a subcommand: "star|stop". Will do.

Reply:
Replaced with "psi-collector start|stop"